contents/handbook/engineering/developing-locally.md

```diff
@@ -37,6 +37,8 @@ This is what we'll be using in the guide below.
 > It is also technically possible to run PostHog in Docker completely, but syncing changes is then much slower, and for development you need PostHog dependencies installed on the host anyway (such as formatting or typechecking tools).
 > The other way around – everything on the host, is not practical due to significant complexities involved in instantiating Kafka or ClickHouse from scratch.
 
+> [Podman](https://podman.io/) and [`podman-compose`](https://github.com/containers/podman-compose) can be used as alternatives to Docker and `docker compose` respectively to allow rootless containers. The commands used in this documentation are analogous to the commands used with Podman (e.g. `podman-compose -f docker-compose.dev.yml up` instead of `docker compose -f docker-compose.dev.yml up`)
```

Comment:
I would maybe drop this paragraph, since introduction of another tool makes the guide a bit less straightforward, and I suppose those who are into `podman` already know what it is about. Definitely makes sense to keep the `sudo`-related mention below.